src/amqp10_client.erl

```diff
@@ -418,11 +428,22 @@ parse_tls_opt(_K, _V, Acc) ->
 to_atom(X) when is_list(X) -> list_to_atom(X).
 to_binary(X) when is_list(X) -> list_to_binary(X).
 
+try_to_existing_atom(L) when is_list(L) ->
+    try list_to_existing_atom(L) of
```

Comment:
FTR, that function [does exist in 19.3](https://erldocs.com/19.0/erts/erlang.html?i=0&search=list_to_e#list_to_existing_atom/1).